src/components/icons/EllipsisHorizontalIcon.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import additionalSVGProps from './Helpers';
 
 type IconProps = import('./types').default;
 
 const EllipsisHorizontalIcon: React.FC<IconProps> = ({
-  width = 24, height = 24, fill = '#000', title = 'EllipsisHorizontal', description = '',
+  width = 24, height = 24, fill = '#000', title = 'EllipsisHorizontal', description = '', context = 'presentational',
```

Comment:
Should this be changed to a `title` of `Ellipsis Horizontal` to be consistent in format with the `Ellipsis Vertical` title below?